src/app/core/services/auth.service.ts

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { inject, Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable, tap } from 'rxjs';
+import { Router } from '@angular/router';
+import * as decoder from 'jwt-decode';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AuthService {
+  private apiUrl = 'http://localhost:3000/auth';
```

Comment:
Put this information on an environment file.  [https://angular.dev/tools/cli/environments#configure-environment-specific-defaults](url)